tests/waveforms/test_cbc_waveforms.py

```diff
@@ -225,16 +225,16 @@ def test_phenom_d(
     hc_torch = hc_torch[torch_mask]
 
     assert np.allclose(
-        1e21 * hp_lal_data.real, 1e21 * hp_torch.real.numpy(), atol=2e-4
+        1e21 * hp_lal_data.real, 1e21 * hp_torch.real.numpy(), atol=3e-4
```

Comment:
I could not make the tests pass without bumping the threshold a bit, which is strange given that spline interpolation should work better than linear. Any thought whether tweaking the kx, sx values can help? CC @wbenoit26

Reply:
Honestly, I might steer clear of using the spline interpolation for this application for now. There are significant deviations from scipy's interpolation if you interpolate too close to the edges of your input range. This is workable for something like the Q-transform, where we can crop out those effects, but that's not something that you can do here. I'd be curious to see if the match is better for values well within the range, though.

Reply:
OK. I'll leave this marked as draft until the sanity is met.